pkg/glusterfs/controllerserver.go

```diff
@@ -133,17 +124,23 @@ func validateThinArbiter(req *csi.CreateVolumeRequest) error {
 	return nil
 }
 
-func addThinArbiter(req *api.VolCreateReq, thinArbiter string) error {
+func addThinArbiter(volReq *api.VolCreateReq, req *csi.CreateVolumeRequest) error {
+
+	if err := validateThinArbiter(req); err != nil {
+		return err
+	}
 
+	thinArbiter := req.Parameters["arbiterPath"]
```

Comment:
Can we make this key a constant ?

Reply:
currently, all the keys we are parsing from SC are not constants https://github.com/gluster/gluster-csi-driver/blob/master/pkg/glusterfs/controllerserver.go#L85

if its required i can do that